backend/Tasque.DAL/Migrations/DataContextModelSnapshot.cs

```diff
@@ -34,7 +34,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.HasIndex("TasksId");
 
-                    b.ToTable("AttachmentTask", (string)null);
```

Comment:
https://github.com/BinaryStudioAcademy/bsa-2022-tasque/pull/225/files#diff-768a9f8593b2787c8140a407517a127d9e1d1d512b13f17621fde11d43856d68

please contact @DaniilPoiarkov regarding this change and find an agreement on reasoning behind this change